src/App.tsx

```diff
@@ -11,6 +11,8 @@ const App = (props: AppProps) => {
         <main className="app veilarbdetaljerfs">
             <StoreProvider fnr={props.fnr}>
                 <Nokkelinfo />
+                <div className='mainGrid'>
```

Comment:
Glemte denne! :)

Reply:
Tusen takk for at du la merke til den! :D